### PR TITLE
Fix warning when opening an executable without any dt_dynamic information

### PR DESCRIPTION
**Pull request by ghost**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

* Fix 1 coverity error (CID 348997)
* Fix warning when the binary as no dt_dynamic information (tiny elf)
* **Enable travis CI on all branches**

**Test plan**

```
rizin test/bins/elf/analysis/tiny.elf
rizin test/bins/elf/analysis/tiny1
rizin test/bins/elf/analysis/tiny1@invalid_addr
```

**Closing issues**

...

### CODE REVIEW

**Review by XVilka — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,6 @@
...
 conditions: v1
 
-branches:
-  only:
-  - stable
-  - dev
-  - /^release-.*$/
-  - /^prerelease-.*$/
-  - /^travis-.*$/
-  - /^import-.*$/
```
Comment (wargio) [inline .travis.yml:11]: i think there was a reason why we had this, which is to avoid travis to run always.

Comment (wargio) [reply .travis.yml:11]: @XVilka

Comment (ghost) [reply .travis.yml:11]: https://im.rizin.re/rizinorg/pl/6aunbdp58tyejmspmafepq491c

Comment (XVilka) [reply .travis.yml:11]: I suggested to do that specifically. Travis is even faster than GitHub Actions nowadays.

**Pull request merged**